OpenRA.Mods.Common/UtilityCommands/ImportLegacyMapCommand.cs

```diff
@@ -9,33 +9,344 @@
 #endregion
 
 using System;
+using System.Collections.Generic;
 using System.IO;
+using System.Linq;
+using System.Text;
+using OpenRA.Graphics;
+using OpenRA.Mods.Common.FileFormats;
+using OpenRA.Mods.Common.Traits;
+using OpenRA.Primitives;
 
 namespace OpenRA.Mods.Common.UtilityCommands
 {
-	class ImportLegacyMapCommand : IUtilityCommand
+	public class ImportLegacyMapCommand : IUtilityCommand
```

Comment:
Not sure how to do that. I get an error upon execution when making this `abstract`:

> System.MissingMethodException: Default constructor not found